README.md

```diff
@@ -26,6 +26,10 @@ Please check it reguarly.
 The following instructions take you through getting the most basic setup working.
 We'll explain what actually happens below the hood, and how to configure things afterwards.
 
+### Update hosts file
+
+As the website is expected to run at `local.exercism.io`, you should update your hosts file to have `local.exercism.io` resolve to `127.0.0.1` (localhost).
```

Comment:
This should actually be supported automatically by the DNS. But I might have configured it wrong. I'm happy for this to go in here for now either way.

Reply:
It is? I can check that in a sec

Reply:
It doesn't work for me without the hosts file update.

Reply:
Annoying! :)